src/store/actions/hardware-wallet/getLedgerAccounts.js

```diff
@@ -5,7 +5,16 @@ export const getLedgerAccounts = async (
   { network, walletId, asset, walletType, startingIndex, numAccounts }
 ) => {
   const { client, networkAccounts } = getters
-  const _client = client(network, walletId, asset, walletType, 0, false)
+  const _client = client(
+    {
+      network,
+      walletId,
+      asset,
+      walletType,
```

Comment:
Hmm, isn't the getter now take `accountId` instead of type and index?

Reply:
Hi good point, but you can still send the wallet type if you don't send the accountId, BTW I removed this call because we don't need it
input parameters:
```
{
      network,
      walletId,
      asset,
      accountId,
      useCache = true,
      walletType = 'default',
      index = 0
    }
```

then we check for the default values 
```
const account = accountId ? getters.accountItem(accountId) : null
const accountType = account?.type || walletType
const accountIndex = account?.index || index
```